src/main/java/io/specto/hoverfly/junit/dsl/matchers/RequestMatcher.java

```diff
@@ -0,0 +1,14 @@
+package io.specto.hoverfly.junit.dsl.matchers;
+
+
+import io.specto.hoverfly.junit.core.model.FieldMatcher;
+
+public interface RequestMatcher {
+
+
+
+    FieldMatcher getFieldMatcher();
```

Comment:
You could have a list of matchers like this:

```java
body(exactlyMatches(foo), matchesJson(bar))
```

In which case each element of this list should be merged into a single fieldmatcher - is this what will happen?